lib/condorSecurity.py

```diff
@@ -215,7 +292,25 @@ def __init__(self, filter=None):
 # the context and feature are related
 # to the Condor protocol handling
 class ProtoRequest(SecEnvRequest):
```

Comment:
Should the inherited class be mentioned in the docstring?

Reply:
If you mean the parent, it's in the signature